plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -278,7 +284,11 @@ internal class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
             unregisterAnimators(this, cancelAnimators = false)
           }
           if (runningAnimatorsQueue.isEmpty()) {
-            performMapJump(cameraOptionsBuilder.build())
+            // TODO handle anchor update manually, to be fixed in core to obtain from CameraOptions
```

Comment:
can we have a ticket tracking it in this repo, so that it can be picked up when the fix in core is ready?

Reply:
https://github.com/mapbox/mapbox-maps-android/issues/119